IFTPipeline.jl/src/feature-extraction.jl

```diff
@@ -90,3 +93,93 @@ function extractfeatures(;
     serialize(joinpath(output, "floe_props.jls"), props)
     return nothing
 end
+
+function extractfeatures_single(;
```

Comment:
Why use a separate name for this function? Can it just be named `extractfeatures`? Because it has a different signature than the other one, multiple dispatch can take care of this.

Reply:
It has a different name to make it match with the command line name. I'm open to changing this.